includes/class-mysql.php

```diff
@@ -30,7 +30,7 @@ function yourls_db_connect() {
         $dbhost = sprintf( '%1$s;port=%2$d', $dbhost, $dbport );
     }
 
-    $charset = yourls_apply_filter( 'db_connect_charset', 'utf8' );
+    $charset = yourls_apply_filter( 'db_connect_charset', 'utf8mb4' );
```

Comment:
Ark, breaking change, right?

Reply:
Not breaking in the sense of "something that was working previously will stop working", that's what the upgrade function is there for. I think that's a typical case for [minor version](https://semver.org/#spec-item-7) (that includes "patches" changes) hence 1.8